src/framework/preferences/js/ArrowScrolling.js

```diff
@@ -77,6 +83,16 @@ var fluid_3_0_0 = fluid_3_0_0 || {};
             "beforeReset.resetPanelIndex": {
                 listener: "{that}.applier.fireChangeRequest",
                 args: {path: "panelIndex", value: 0, type: "ADD", source: "reset"}
+            },
+            "afterScroll.setPanelIndex": {
+                changePath: "panelIndex",
+                value: {
+                    expander: {
+                        funcName: "fluid.prefs.arrowScrolling.getClosesPanelIndex",
+                        args: "{that}.dom.panels"
+                    }
+                },
+                source: "manualScroll"
```

Comment:
This source name doesn't seem too accurately descriptive - "manualScroll" might equally well describe swiping or sliding the entire panel or its scrollbar. Perhaps something like panelIndexScroll or arrowScroll?

Reply:
@amb26 actually this is to handle the case where the panel was scrolled by swiping, sliding, or using the keyboard to change the scroll position. But the scroll event is also triggered when scrolling with the arrow buttons too.